makefiles/Base.toml

```diff
@@ -47,22 +51,17 @@ script = "echo Cannot build for Mac from Linux && false"
 
 [tasks.build-dev-musl.linux]
 dependencies = ["setup-musl"]
-command = "cargo"
-args = [
-    "build",
-    "--bin", "fluvio",
-    "--target=x86_64-unknown-linux-musl",
-]
+script = """
+cargo build --bin fluvio --target=x86_64-unknown-linux-musl
```

Comment:
This is ok for now.  But this is where we could collapse into a single script/rule rather than duplicating logic.  @tjtelan Please add this to where we can clean up.

Reply:
We will be adding more targets pretty soon